google/cloud/bigtable/rpc_backoff_policy.cc

```diff
@@ -30,7 +30,8 @@ ExponentialBackoffPolicy::ExponentialBackoffPolicy(
 }
 
 std::unique_ptr<RPCBackoffPolicy> ExponentialBackoffPolicy::clone() const {
-  return std::unique_ptr<RPCBackoffPolicy>(new ExponentialBackoffPolicy(*this));
+  return std::unique_ptr<RPCBackoffPolicy>(
+      new ExponentialBackoffPolicy(initial_delay_, maximum_delay_));
```

Comment:
Possible follow-up note:  It seems to me that copy construction/assignment for these "clone-able" classes is an attractive nuisance (they don't support equality after all), and that we'd be well served trying to delete that capability.